docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -311,8 +310,18 @@ These steps can be performed from within Buildkite's interface. From Buildkite's
 
 <TabItem value='buildkite-semgrep'>
 
+The following configuration creates a CI job that runs scans according to the products you have enabled in Semgrep AppSec Platform. The provided environment variables are commonly needed to correctly configure scans from Buildkite.
 
-The following configuration creates a CI job that runs scans depending on what products you have enabled in Semgrep AppSec Platform.
+This file configures two alternative command paths, one for full scans, and one for diff-aware scans. The latter is used for pull or merge requests.
```

Comment:
```suggestion
This file configures two paths: one for full scans, and one for diff-aware scans. The latter is used for pull or merge requests.
```

Reply:
For this I realized I should probably use "mutually-exclusive command steps". Command is a step type in BK, and it's important for users to understand that there are two jobs but only one will ever be used for a given run if it's properly configured.